.github/ISSUE_TEMPLATE/Decision-Reviews-Bug-Template.md

```diff
@@ -1,9 +1,10 @@
 ---
-name: Decision Reviews Bug Issue
-about: Bug issues for the Decision Reviews team
+name: Decision Reviews Bug Template
+about: Bug template for the Decision Reviews team
 title: ''
-labels: bug, Decision-Reviews-Team, needs-grooming, needs-refinement
+labels: bug, Decision-Reviews-Team, team-DRAGONS, needs-refinement
 assignees: ''
+projects: ["department-of-veterans-affairs/1434"]
```

Comment:
Since our project board is set up to pull in anything with the "team-DRAGONS" label, we shouldn't need to explicitly set the projects key in these templates.